yarn.lock

```diff
@@ -560,21 +560,29 @@
   dependencies:
     "@sinonjs/commons" "^1.7.0"
 
+"@sofie-automation/blueprints-integration@npm:@tv2media/blueprints-integration@46.0.2":
+  version "46.0.1"
+  resolved "https://registry.yarnpkg.com/@tv2media/blueprints-integration/-/blueprints-integration-46.0.1.tgz#e344f31e3fcc48d582914ebc33378a626cebd0e6"
+  integrity sha512-y0ITMUH0JrDligG9X+mWsvVnnGYmZJTrx5iE2XVsHiQ1xQTeBrqDyoxOag2lpbUIF8HI7mXCQkGfk7AYvQLopQ==
```

Comment:
This is confusing. 46.0.2 is broken on NPM, but looks like even with 46.0.1 we have the dependencies we want and all builds